serverless-workflow-callback-quarkus/src/main/resources/application.properties

```
@@ -1,5 +1,7 @@
 kafka.bootstrap.servers=localhost:9092
 
-
 mp.messaging.incoming.move.connector=smallrye-kafka
-mp.messaging.incoming.move.value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
\ No newline at end of file
+mp.messaging.incoming.move.value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
+
+kogito.persistence.type=jdbc
+quarkus.datasource.db-kind=postgresql
```

```suggestion
%persistence.quarkus.datasource.db-kind=postgresql
```

Maybe adding in the maven profile a property like `quarkus.profile=persistence` as well? 🤔

---

I guess is not really necessary, it will be ignored if not using persistence anyway.